src/tiledb/cloud/soma/ingest.py

```diff
@@ -25,6 +25,7 @@ def register_dataset_udf(
     namespace: Optional[str] = None,
     config: Optional[Mapping[str, Any]] = None,
     verbose: bool = False,
+    **kwargs,
```

Comment:
Adding a catch-all `**kwargs` to all of our distributed functions allows them to ignore unexpected keyword arguments. I think we might make this standard?